### PR TITLE
GITOPS-2269 - updated 4.8 release notes

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise- 4.8

JIRA issue:
https://issues.redhat.com/browse/GITOPS-2269

Preview pages: 

SME+QE review: @wtam2018 
Peer-review: @gabriel-rh

### CODE REVIEW

**Review by wtam2018 — COMMENTED** [review]
LGTM

**Review by gabriel-rh — COMMENTED** [review]
Looks good! just some suggestions

File: modules/gitops-release-notes-1-3-6.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+= Release notes for {gitops-title} 1.3.6
+
+{gitops-title} 1.3.6 is now available on {product-title} 4.7 and above.
+
+[id="fixed-issues-1-3-6_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* In {gitops-title}, improper access control allows admin privilege escalation link:https://access.redhat.com/security/cve/CVE-2022-1025[(CVE-2022-1025)]. This update fixes the issue.
```
[the remark this answers is not in the document]
Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-3-6.adoc:15]: It is consistent

**Review by gabriel-rh — APPROVED** [review]
/lgtm

Reviewed without a preview available, but source looks good

**Pull request merged**